editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -483,11 +478,17 @@ const NewCanvasControlsInner = (props: NewCanvasControlsInnerProps) => {
           {when(isFeatureEnabled('Canvas Strategies'), <GuidelineControls />)}
           <OutlineHighlightControl />
           {when(
-            isFeatureEnabled('Canvas Strategies'),
+            isCanvasStrategyInSelectOrSelectLiteMode(props.editor.mode),
             <>{strategyControls.map((c) => React.createElement(c.control, { key: c.key }))}</>,
           )}
         </>,
       )}
     </div>
   )
 }
+
+function isCanvasStrategyInSelectOrSelectLiteMode(mode: Mode): boolean {
+  return (
+    (isFeatureEnabled('Canvas Strategies') && mode.type === 'select') || mode.type === 'select-lite'
```

Comment:
It could be how it reads just to me, but this condition is not what I'd expect from the function name.